spec/features/manage_cmr/subscriptions/index_subscriptions_spec.rb

```diff
@@ -85,6 +84,7 @@
     context 'when viewing the index page with full permissions' do
       before do
         allow_any_instance_of(SubscriptionPolicy).to receive(:edit?).and_return(true)
+        allow_any_instance_of(SubscriptionPolicy).to receive(:update?).and_return(true)
```

Comment:
I think that this should either be index, edit, destroy, and create or index, and update.

Reply:
I think this context block should be pulled out to be at the same indentation level as line 34. Then you don't have to mock any of the policy methods, because they are being granted in the first before :all block.